src/Exception/CompilerException.php

```diff
@@ -49,7 +49,7 @@ public static function unknownRequestType($request)
     {
         $code = self::UNKNOWN_REQUEST_TYPE;
         $data = array('request' => $request);
-        $message = 'Only get, delete, and set queries are supported at the moment.';
+        $message = 'Only get, delete, set, and insert queries are supported at the moment.';
```

Comment:
We support a variety of top-level functions now--and we're likely to add more in the future. This message may be confusing to people who know about the "sum" and "count" queries, for example.